src/webui.c

```diff
@@ -175,6 +215,12 @@ process_request(struct evhttp_request *req, void *arg)
         return;
     }
 
+    if (strstr(url, "/worker/") != NULL)
```

Comment:
As you're simply skipping 8 chars (in send_json_worker), you should really strncmp here instead of strstr. 

You'll also want to change the other strstr's above in this function to strncmp as otherwise a worker name of "stats" or "workers" will match above and never get here.

Reply:
changed all priors to strncmp's